### ISSUE TITLE
Neoformat writes to buffer on formatter error

### ISSUE THREAD

**Issue by FuzzOli87** [sbdchd/neoformat#164]
Hello,

I'm running into an issue with prettier-eslint-cli. Just updated my modules so I'm digging around a bit more.

However, there is a recurring issue I see and that is that when neoformat fails to run the formatter, it writes the error to the file.

These are my current configurations:
```
let g:neoformat_javascript_prettiereslint = {
  \ 'exe': 'prettier-eslint',
  \ 'args': ['--stdin'],
  \ 'stdin': 1
  \ }
let g:neoformat_enabled_javascript=['prettiereslint']
```

**Comment by sbdchd** [comment]
If you `set verbose=1`, and run `:Neoformat`, what exit code does it print out?

**Comment by FuzzOli87** [comment]
![image](https://user-images.githubusercontent.com/12246915/40930554-9c17e7c0-67ed-11e8-91db-782847250193.png)

Here is the tail-end of the output @sbdchd

**Comment by sbdchd** [comment]
yeah, probably should make the logging more explicit. Pretty sure that is an exit code of `1`, while valid exit codes being `[0]`.

**Comment by pxue** [comment]
@FuzzOli87 did you ever figure this out?

Seems like prettier-eslint is reading the file from stdin and it's having some issue loading up the eslintrc usually read from the file location.. hence the "No parser and no filepath given"
